pkg/limits/ingest_limits.go

```diff
@@ -279,40 +282,6 @@ func (s *IngestLimits) running(ctx context.Context) error {
 		// stop
 		case err := <-s.lifecyclerWatcher.Chan():
 			return fmt.Errorf("lifecycler failed: %w", err)
-		default:
```

Comment:
I assume this will come back in a later PR?

Reply:
Yes this will come back when we do the replays from the queue.